test/schema/home/home_page_artwork_modules.js

```diff
@@ -109,6 +116,7 @@ describe('HomePageArtworkModules', () => {
           'followed_galleries',
           'saved_works',
           'current_fairs',
+          'followed_artist',
           'related_artists',
```

Comment:
Good point, this does break that concept of being able to provide an order beforehand.